### DISCUSSION TITLE
module 'cryptography.utils' has no attribute 'bit_length'

### DISCUSSION THREAD

**Discussion by victorg737** [ktbyers/netmiko#2115] [Q&A] [answered]
using python 3.8 and netmiko 3.3.2 and having issues connecting with HP Procurve switches, have tried a few different models. I can connect to cisco gear just fine. 

Here is the error I am getting:
AttributeError: module 'cryptography.utils' has no attribute 'bit_length'

I saw in posts it was fixed in the development in this post
https://github.com/ktbyers/netmiko/issues/2057

assuming its still not fixed in general realease, how to I install the develop branch?

**Comment by ktbyers** [comment, marked as answer]
Probably the easiest fix is to just use a downgraded version of the `cryptography` library.

`pip install cryptography==2.9.2`

Alternatively, you should be able to `git clone` the develop branch and `pip install -e .` that repository. There is also a way you can directly install the `develop` branch via `pip` (the syntax is just a bit cumbersome).


Regards, Kirk
